tests/unit/lms/views/error_test.py

```diff
@@ -1,167 +1,140 @@
 from unittest import mock
```

Comment:
This whole file looks nuts as a diff, but open it up and it makes way more sense: https://github.com/hypothesis/lms/blob/f8830bb9958e34eed23c54343aeaedaeaed6a978/tests/unit/lms/views/error_test.py